backend/satellite_tools/repo_plugins/deb_src.py

```diff
@@ -247,8 +247,8 @@ def get_package_list(self):
 class ContentSource:
 
     def __init__(self, url, name, insecure=False, interactive=True, yumsrc_conf=None,
-                 org="1", channel_label="", no_mirrors=False, ca_cert_file=None,
-                 client_cert_file=None, client_key_file=None):
+                 org="1", channel_label="", no_mirrors=False, http_headers=None,
```

Comment:
Just a note, based on a quick `grep` I don't think it causes problems here but I think it could be worth sharing anyway.

By default, Python function* arguments can be passed by position or by keyword. Once the first argument is passed by keyword, no positional arguments are accepted. Changing the order of the arguments can (silently) change behavior when callers call the function with positional arguments. For this reason, new arguments are normally added to the end of the function signature. 
However, calling functions* using positional arguments is not good practice for functions with many arguments anyway since it makes the code harder to read.

Keyword-only calling can be enforced by adding a `*` to the function arguments. Any argument after it **must** be called by keyword:
 ```
 In [7]: def foo(*, bar=None, baz=None):
   ...:     print(bar, baz)
   ...: 

In [8]: foo()
None None

In [9]: foo(1,2)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-9-49e36bfb7156> in <module>
----> 1 foo(1,2)

TypeError: foo() takes 0 positional arguments but 2 were given
```

For functions that enforce keyword-only calling, new keyword arguments can be added anywhere since the order is not important. Functions that don't enforce keyword-only calling should be careful.

*: strictly speaking it is any callable object, not just functions

Reply:
Thanks for the explanation Alex. I will adapt this change to add the new argument to the end of the list.